test/integration/auth_webhook_test.go

```diff
@@ -206,9 +206,9 @@ func TestAuthWebhook(t *testing.T) {
 			}
 		}))
 
-		var authorizedTTLSec uint64 = 1
+		authorizedTTLSec := 1 * time.Second
```

Comment:
We can drop unit-specific suffix now because we replace the type with time.Duration.
```suggestion
		authorizedTTL := 1 * time.Second
```